tests/base.py

```diff
@@ -230,3 +244,14 @@ def minimum_record_count_by_stream(self):
         record_counts["vendors"] = 26
 
         return record_counts
+
+    def dt_to_ts(self, dtime):
```

Comment:
Raising an exception after this for loop would prevent the test from getting back a None and failing in a confusing way in the case where the datetime format is unaccounted for.